### ISSUE TITLE
Infoboxer intro includes single math formulas

### ISSUE THREAD

**Issue by phlegx** [molybdenum-99/infoboxer#68]
Hi!

If I want to fetch for e.g. this site:

```ruby
paragraphs = Infoboxer.wikipedia('de').get('Energiedosis').intro
```

**I get the math formula as second paragraph. Why?**

Same here:

```ruby
paragraphs = Infoboxer.wikipedia('es').get('Potencia_(física)').intro[2].to_s
```

```
 => "La potencia instantánea es el valor límite de la potencia media cuando el intervalo de tiempo
Δt se aproxima a cero. En el caso de un cuerpo de pequeñas dimensiones: {\\Delta t} = \\m
athbf{F}\\cdot \\mathbf{v} ||left Donde" 
```

If the template starts with **{{** and checks all variables with `@context.eat_matched?('}}')` as end, then it stops in the middle of the formula after `\Delta\mathbf{r`.

```
{{ecuación|
<math>P(t) = \lim_{\Delta t\rightarrow 0} \frac{\ W}{\Delta t}\ =
\lim_{\Delta t\rightarrow 0} \mathbf{F}\cdot\frac{\Delta\mathbf{r}}{\Delta t} =
\mathbf{F}\cdot \mathbf{v}</math>
||left}}
```

**My opinion: tag `<math>` should remain in the result so that can be handled in HTML!**

**Comment by zverok** [comment]
Definitely a bug. Looking into it.

**Comment by zverok** [comment]
Hey.
In fact, two problems here: first is `<math>` rendering to text, and second is mix of templates and `<math>`, erroneously parsed. The second one is fixed by #70, and the first I've fixed too, now `Math` node is surrounded by `<math>` tag while rendering to text.

Everything is in `develop` branch, please confirm it works for you as expected.
